runtime/eoapi/stac/eoapi/stac/config.py

```diff
@@ -1,35 +1,39 @@
 """API settings."""
 
 from functools import lru_cache
-from typing import Optional
+from typing import List, Optional
 
 import pydantic
-from stac_fastapi.api.models import create_get_request_model, create_post_request_model
-from stac_fastapi.extensions.core import (
-    ContextExtension,
-    FieldsExtension,
-    FilterExtension,
-    QueryExtension,
-    SortExtension,
-    TokenPaginationExtension,
-)
-from stac_fastapi.pgstac.extensions.filter import FiltersClient
-from stac_fastapi.pgstac.types.search import PgstacSearch
 
 
 class _ApiSettings(pydantic.BaseSettings):
     """API settings"""
 
     name: str = "eoAPI-stac"
     cors_origins: str = "*"
+    cors_methods: str = "GET,POST,OPTIONS"
     cachecontrol: str = "public, max-age=3600"
     debug: bool = False
 
+    extensions: List[str] = [
+        "filter",
+        "query",
+        "sort",
+        "fields",
+        "pagination",
+        "context",
+    ]
```

Comment:
Defaults extensions (maybe we should remove the `context` extension because it needs to be enabled in the database)

Reply:
Any reason we're not adding the transaction extension by default ?

Reply:
Ideally the transaction extension should require auth, and because we are also shipping the custom runtime with the ingestion pipeline I don't think we need to add in as default